src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/Directory.Build.targets

```diff
@@ -1,32 +1,42 @@
 <Project>
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)..\'))" />
```

Comment:
Looks like same issue also exists in the below `Directory.Build.props` which was copied from this one. Could you please fix that one too?

https://github.com/shyamnamboodiripad/extensions/blob/main/src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Directory.Build.targets#L2